### PR TITLE
Kdegear update to 24.05.1

### PR DESCRIPTION
**Pull request by Luciogi**
#### Testing the changes
- I tested the changes in this PR: **YES**>

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-GLIBC)
- armv7l Glibc

### CODE REVIEW

**Comment by Luciogi** [comment]
- [ ] Test kdiff for armv6
https://github.com/void-linux/void-packages/pull/50887/files#diff-2d2ebfd3f327db8d4e3a2f65920a6adb90897d72f5ea3697066b58914dd48e5c

**Comment by Luciogi** [comment]
@sgn 
review

File: srcpkgs/elisa/template [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
 homepage="https://kde.org/applications/en/multimedia/org.kde.elisa"
 distfiles="${KDE_SITE}/release-service/${version}/src/elisa-${version}.tar.xz"
-checksum=08218ba739ea0ace82bba6aa98575814dcfdf69feb61596398f5d337c2956c0f
+checksum=55ed76a676af42c3af455f64a7f63f51a3fbaa619c92d345be6708ad5cdc77e2
+
+do_build() {
+	cmake -B build -G Ninja \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib
```
Comment (sgn) [inline srcpkgs/elisa/template:29]: This is bad!

File: srcpkgs/kasts/template [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Template file for 'kasts'
 pkgname=kasts
-version=24.02.2
+version=24.05.1
 revision=1
 build_style=cmake
+build_helper=qemu
```
Comment (sgn) [inline srcpkgs/kasts/template:6]: Nope, add `qt6-declarative-host-tools` into `hostmakedepends` instead.

File: srcpkgs/elisa/template (hunk 1 above)
Comment (Luciogi) [reply srcpkgs/elisa/template:29]: oh I forgot to remove

File: srcpkgs/kasts/template (hunk 2 above)
Comment (Luciogi) [reply srcpkgs/kasts/template:6]: is this valid for elisa too ?

File: srcpkgs/elisa/template [hunk 3]
```diff
@@ -1,19 +1,23 @@
 # Template file for 'elisa'
 pkgname=elisa
-version=24.02.2
-revision=2
+version=24.05.1
+revision=1
 build_style=cmake
+build_helper=qemu
```
Comment (sgn) [inline srcpkgs/elisa/template:6]: You are doing it wrong again. It only need `qt6-declarative-host-tools` in `hostmakedepends`.

File: srcpkgs/kasts/template (hunk 2 above)
Comment (sgn) [reply srcpkgs/kasts/template]: valid for everything.

**Pull request closed without merging**